script.js

```diff
@@ -0,0 +1,177 @@
+//   },
+//   {
+//     name: 'yclogo',
+//     logo: 'sponsers-img/5e5816226dfe7b54f5cef482_yclogo.png',
+//   },
+//   {
+//     name: 'L2 VENTURES',
+//     logo: 'sponsers-img/5e5816226dfe7b896acef481_l2.png',
+//   },
+//   {
+//     name: 'hackvc',
+//     logo: 'sponsers-img/5e58162203fc89e9aa5acc3a_hackvc.webp',
+//   },
+//   {
+//     name: '',
+//     logo: '',
+//   },
+//   {
+//     name: 'Northzone',
+//     logo: 'sponsers-img/61522595be718075aca25e5c_northzone.png',
+//   },
+// ];
+
+// const learn = [{
+//   image: 'images/program_icon_04.png',
+//   title: 'NO UPFRONT COST',
+//   discretion: 'You owe us nothing until you get a job —no matter what country you’re in.',
+
+// }, {
+//   image: 'images/program_icon_03.png',
+//   title: 'COLLAB LEARNING',
+//   discretion: 'Learn through pair programming, group projects,and project reviews.',
+
+// }, {
+//   image: 'images/program_icon_01.png',
+//   title: 'GLOBAL CHANCE',
+//   discretion: 'Connect with global jobs after gaining experience working on a remote team.',
+// }, {
+//   image: 'images/program_icon_05.png',
+//   title: 'SUPPORT AND LIABLE',
+//
+// },
+// ];
+// const story = [{
+//   name:"kevin Mwangi",
+//   image:"images/Screenshot (17).png"
+//   ,link:"https://www.microverse.org/blog/a-career-with-microsoft-how-microverse-helped-pave-the-way"
+// },
+// {name:"Gabriela Cruz Olivera",
+// image:"images/Screenshot (11).png",
+// link:"https://www.microverse.org/blog/join-the-global-job-market-as-a-web-developer"}
+// ];
+
+const team = document.querySelector('.team-box');
+// const spons = document.querySelector('.backed-by');
+// const box = document.querySelector('.main-big-box');
+// const success = document.querySelector(".success-img");
+
+function boredmembers() {
+  teammembers.forEach((teammembers) => {
+    team.innerHTML += `
+      <div class="mobile-version">
+      <div class="team-profile">
+                <div class="image-box">
+                    <img class="square-img" src="/images/Screenshot (12).png" alt="">
+                <img class="team-image"src="${teammembers.image}" alt="${teammembers.name} image">
+          </div>
+          <div class="team-name"><h4>${teammembers.name}</h4>
+            <p>${teammembers.profession}</p></div>
+            </div>
+            <p>${teammembers.discretion}</p></div>
+
+            <div class="desktop-version">
+      <div class="team-profile">
+                <div class="image-box">
+                    <img class="square-img" src="images/Screenshot (12).png" alt="">
+                <img class="team-image"src="${teammembers.image}" alt="${teammembers.name} image">
+          </div>
+          <div class="minebox">
+          <div class="team-name"><h4>${teammembers.name}</h4>
+            <p>${teammembers.profession}</p>
+            </div>
+            <p>${teammembers.discretion}</p></div></div></div>
+    `;
+  });
+}
+boredmembers();
```

Comment:
- [ ] I kindly suggest that you wrap this function is a window.onload function so that it will be executed only when the web page has completely loaded all content

Reply:
Fixed.